examples/stats/plot_sensor_regression.py

```diff
@@ -5,73 +5,43 @@
 
 Predict single trial activity from a continuous variable.
 A single-trial regression is performed in each sensor and timepoint
-individually, resulting in an Evoked object which contains the
-regression coefficient (beta value) for each combination of sensor
-and timepoint. Example also shows the T statistics and the associated
-p-values.
-
-Note that this example is for educational purposes and that the data used
-here do not contain any significant effect.
-
-(See Hauk et al. (2006). The time course of visual word recognition as
-revealed by linear regression analysis of ERP data. Neuroimage.)
+individually, resulting in an Evoked object which contains the regression
+coefficient (beta value) for each combination of sensor and timepoint. Example
+shows the regression coefficient; the t and p values are also calculated
+automatically.
+
+Here, we repeat a few of the analyses from Dufau et al. (2014) by accessing
+the metadata object, which contains word-level information about various
+psycholinguistically relevant features of the words for which we have EEG
+activity.
+
+For the general methodology, see e.g. Hauk et al. (2006). The time course of
```

Comment:
Better to do proper references here with `[1]_` and `.. [1]` in a `References` section